db/verify/functions/receive_applications.sql

```diff
@@ -0,0 +1,7 @@
+-- Verify ccbc:functions/receive_applications on pg
+
+BEGIN;
+
```

Comment:
We can just delete verify files as we aren't using them anymore, same applies to the rest of the `verify` files